src/Symfony/Component/Validator/Tests/Constraints/IbanValidatorTest.php

```diff
@@ -109,6 +109,7 @@ public static function getValidIbans()
             ['MU17 BOMM 0101 1010 3030 0200 000M UR'], // Mauritius
             ['MD24 AG00 0225 1000 1310 4168'], // Moldova
             ['MC93 2005 2222 1001 1223 3M44 555'], // Monaco
+            ['MN14 0005 0051 6384 7716'], // Mongolia
```

Comment:
Should other IBAN formats be added to the data provider?

Reply:
I see if I can find example IBANs for the other added countries

Reply:
```
FK12SC987654321098
OM040280000012345678901
YE09CBKU0000000000001234560101
```

Reply:
added, thanks 👍